src/django_twc_toolbox/crud/templatetags/neapolitan.py

```diff
@@ -48,7 +49,7 @@ def object_detail(object: models.Model, view: CRUDView):
     def iter() -> Generator[tuple[str, str], None, None]:
         for f in fields:
             mf = object._meta.get_field(f)
-            yield (mf.verbose_name, str(getattr(object, f)))
+            yield (cast(str, mf.verbose_name), str(getattr(object, f)))  # type: ignore[union-attr]
```

Comment:
This example might be what scares people off of typing and mypy 🤔

Reply:
Oh absolutely, it would scare me off but I'm pigheaded with this stuff and refuse to give in.

I _think_ `django-stubs` is wrong here, and I spent sometime digging in to why, but I don't have the energy to chase it all down and write a decent enough issue in the repo.